### PR TITLE
Improve consistency of NavigationCardStack animations.

### PR DESCRIPTION
**Pull request by domchristie**
Currently there’s an inconsistency between the animations used in `NavigationAnimatedView` (`spring`) and those in `NavigationCardStack` (`timing`), which is noticeable when switching between the two implementations.

By removing the `_applyAnimation` method, the `NavigationAnimatedView` will simply use its [default `applyAnimation`](https://github.com/facebook/react-native/blob/6c22a2174e733d536a12f47a2da4f84329ac3023/Libraries/NavigationExperimental/NavigationAnimatedView.js#L56-L67), making the animation styles the same.

**Before** (with `Animated.timing`)
Video: http://quick.as/Yexku8DdJ

**After** (with the default `NavigationAnimatedView` animations)
Video: http://quick.as/qrqbsnj8n

### CODE REVIEW

**Comment by hedgerwang** [comment]
@ericvicenti : This makes sense to me. Could you merge this? (somehow I don't have the permission to merge this)

**Comment by hedgerwang** [comment]
@domchristie : This PR makes sense for now, since we really don't know what the default animation UX should be like in the card stack. In the future, when necessary, we could always added this back.

File: Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js [hunk 1]
```diff
@@ -120,7 +117,6 @@ class NavigationCardStack extends React.Component {
         navigationState={this.props.navigationState}
         renderOverlay={this.props.renderOverlay}
         renderScene={this._renderScene}
-        applyAnimation={this._applyAnimation}
```
Comment (ericvicenti) [inline Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js]: Can we add this.props.applyAnimation in a followup, to allow people to customize it?

**Comment by ericvicenti** [comment]
@hedgerwang, we should be able to ship things by calling to the bot:

**Comment by ericvicenti** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**